ftw/addressblock/contact/browser/contact.py

```diff
@@ -26,6 +25,10 @@
 from zope.interface import Invalid
 
 IS_PLONE_5 = pkg_resources.get_distribution('Products.CMFPlone').version >= '5'
+IS_SUBSITE_INSTALLED = pkg_resources.get_distribution('ftw.subsite')
```

Comment:
@busykoala this will with a `pkg_resources.DistributionNotFound` when `ftw.subsite` is not installed. So it actually fails instead of disabling the feature.

This is best practice for this use case:
```python
try:
    pkg_resources.get_distribution('ftw.subsite')
except pkg_resources.DistributionNotFound:
    IS_SUBSITE_INSTALLED = False
else:
    IS_SUBSITE_INSTALLED = True
```

Reply:
Thanks, good to know that an error will be raised. I implemented the suggested.